ts/index.ts

```diff
@@ -76,7 +78,7 @@ class ElementaryRunner implements CompileOK {
       }),
       require: (lib: string): any => {
         if (this.codeMap[lib]) {
-          return this.codeMap[lib];
+          return Object.freeze(this.codeMap[lib]);
```

Comment:
I am in favor using Object.freeze. But, would you add a comment explaining what the threat is? i.e., what could a student do that warrants the `Object.freeze`?

Reply:
Do you mean a comment in the code? This isn't the first place `Object.freeze` is used.